tools/mason/MasonPublish.chpl

```diff
@@ -78,6 +81,35 @@ proc masonPublish(ref args: list(string)) throws {
       args.append(potentialPath);
     }
 
+    if createReg {
+      var pathReg = '';
+      if args[args.size-1] == '-c' || args[args.size-1] == '--create-registry' then
+        pathReg = args[args.size-2];
+      else if args[args.size-2] == '-c' || args[args.size-2] == '--create-registry' then
+        pathReg = args[args.size-1];
+      try! {
+        if pathReg == 'publish' then throw new owned MasonError('Valid path required ' +
+            'with "mason publish --create-registry"');
+        if !isDir(pathReg) then mkdir(pathReg);
+        else throw new owned MasonError("Registry already exists at %s".format(pathReg));
+        if !isDir(pathReg + '/Bricks') then mkdir(pathReg + '/Bricks');
+        if !isDir(pathReg + '/README.md') then touch(pathReg + '/README.md');
+        if !isDir(pathReg + '/.git') {
+          gitC(pathReg, 'git init -q');
+          gitC(pathReg, 'git add .');
+          commitSubProcess(pathReg, ['git','commit', '-q', '-m',' "initialised registry"']);
+        }
+        writeln("Initialised local registry at %s".format(pathReg));
+        writeln("Add this registry to MASON_REGISTRY environment variable to include it in search path: ");
+        writeln("   export MASON_REGISTRY=${MASON_REGISTRY},%s".format(pathReg));
```

Comment:
Because`MASON_REGISTRY` defaults to `mason-registry|https://github.com/chapel-lang/mason-registry` when unset, the suggested command here will actually override that default and _only_ use the added registry. I suspect most users will still want the default `mason-registry` in their search path, so I am not sure this is the guidance we want to give.

That said, there's not really a succinct way to set `MASON_REGISTRY` to the default registry and then append the new registry to that, so I am not sure what we should recommend.

One idea is to not suggest anything and let the user figure out what must be done. We could also explore options for making this easier in the future.

Reply:
Oh, how about we just print the actual value of MASON_REGISTRY in the export command? e.g.

```
Initialised local registry at foo
Add this registry to your MASON_REGISTRY environment variable to include it in registry search path:

   export MASON_REGISTRY="mason-registry|https://github.com/chapel-lang/mason-registry,foo| /full/path/to/foo"
```